deploy/Dockerfile

```diff
@@ -19,10 +22,25 @@ RUN sed -i "s/scram-sha-256/trust/g" /etc/postgresql/14/main/pg_hba.conf \
    && service postgresql restart \
    && createuser -h localhost -U postgres dbsp
 
+# Install Prometheus
+RUN curl -LO https://github.com/prometheus/prometheus/releases/download/v2.41.0/prometheus-2.41.0.linux-amd64.tar.gz \
+   && tar xvfz prometheus-*.tar.gz \
+   && cd prometheus-* \
+   && mv prometheus /bin/
+
+# Install grafana
+RUN curl -LO https://dl.grafana.com/enterprise/release/grafana-enterprise_9.3.6_amd64.deb \
+   && dpkg -i grafana-enterprise_9.3.6_amd64.deb
+
 ADD ./dbsp_files.tar /database-stream-processor
 ADD ./sql_compiler_files.tar /sql-to-dbsp-compiler
 
-RUN cd /database-stream-processor && ~/.cargo/bin/cargo build --release
+RUN cd /database-stream-processor/pipeline_manager && ~/.cargo/bin/cargo build --release
 RUN cd /sql-to-dbsp-compiler/SQL-compiler && mvn -DskipTests package
 
+# Provision Prometheus data source + DBSP dashboard in Grafana.
+RUN cp /database-stream-processor/deploy/prometheus_datasource.yaml /etc/grafana/provisioning/datasources/
```

Comment:
Every RUN command creates another docker layer. For these three, it's fine to make them a single set of chained commands in one RUN.

Reply:
thanks, fixed